modules/sdpolicy-storage.adoc

```diff
@@ -8,20 +8,21 @@
 
 [id="encrypt-rest-node_{context}"]
 == Encrypted-at-rest OS/node storage
-Master nodes use encrypted-at-rest-EBS storage.
+Control nodes use encrypted-at-rest-EBS storage.
 
 [id="encrypt-rest-pv_{context}"]
 == Encrypted-at-rest PV
 EBS volumes used for persistent volumes are encrypted-at-rest by default.
 
 [id="block-storage_{context}"]
 == Block storage (RWO)
-Persistent volumes are backed by block storage (AWS EBS and Google Cloud persistent disk), which is Read-Write-Once. On an {product-title} base cluster, 100GB of block storage is provided for persistent volumes, which is dynamically provisioned and recycled based on application requests. Additional persistent storage can be purchased in 500GB increments.
+Persistent volumes are backed by block storage (AWS EBS and Google Cloud persistent disk), which is Read-Write-Once. On a standard {product-title} base cluster, 100GB of block storage is provided for persistent volumes, which is dynamically provisioned and recycled based on application requests. Additional persistent storage can be purchased in 500GB increments.
```

Comment:
```suggestion
Persistent volumes (PVs) are backed by AWS EBS and Google Cloud persistent disk block storage, which uses the ReadWriteOnce (RWO) access mode. On a standard {product-title} base cluster, 100 GB of block storage is provided for PVs, which is dynamically provisioned and recycled based on application requests. Additional persistent storage can be purchased in 500 GB increments.
```

Reply:
I've used the "Suggested change" feature in a few places because it can sometimes be easier to show this way. But just a reminder not to use the "Commit suggestion" button; rather, you will need to apply any accepted changes manually. Thanks!

Reply:
Resolved